module/VuFind/sql/pgsql.sql

```diff
@@ -183,10 +183,28 @@ session_id varchar(128),
 data text,
 last_used int NOT NULL default 0,
 created timestamp NOT NULL default '1970-01-01 00:00:00',
+external_session_id varchar(255),
```

Comment:
Oops, they shouldn't have been there in the first place. They were just remains of a broken idea, and have now been removed.